lib/bolt/node/winrm.rb

```diff
@@ -361,9 +361,9 @@ def execute(command, _ = {})
 
       output = @session.run(command) do |stdout, stderr|
         result_output.stdout << stdout
-        @logger.debug { "stdout: #{stdout}" }
```

Comment:
These are subcommands and should be logged at the debug level

Reply:
JSON?

Reply:
no there is a good chance they're not JSON